FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -166,23 +166,39 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </objectDesc>
                          
                         <bindingDesc>
-                            <binding contemporary="partly"></binding>
+                            <binding contemporary="partly">
                             <decoNote xml:id="b1" type="Boards">Two wooden boards.</decoNote>    
                             <decoNote xml:id="b2" type="SewingStations">4</decoNote>
                             <decoNote xml:id="b3" type="bindingMaterial"><material key="wood"></material></decoNote>
+                            </binding>
                         </bindingDesc>
                        
                     </physDesc>
-                    <history><provenance>Scribe <persName>አርሳንዮስ፡</persName> mentioned in <locus target="coloph1"/> and <locus target="coloph2"></locus></provenance></history>
-                    <msPart xml:id="p1"><msIdentifier><msContents>
```

Comment:
You had `<msIdentifier>`, that is you opened the tag, but never closed it. To close an empty tag, add slash at the end `<msIdentifier/>`